var/spack/repos/builtin/packages/py-pysam/package.py

```diff
@@ -39,3 +39,7 @@ class PyPysam(PythonPackage):
     depends_on('py-setuptools', type='build')
     depends_on('py-cython@0.21:', type='build')
     depends_on('bcftools')
+    depends_on('curl')
+
+    # pysam works well with htslib@1.6 but conflicts with htslib@1.7
+    conflicts('htslib@1.7:')
```

Comment:
This doesn't actually work, you need a leading `^` to get it to work. 

Should `py-pysam` have a direct dependency on `htslib`?